test/e2e/specs/editor/blocks/group.spec.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Group', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'can be created using the block inserter', async ( {
+		editor,
+		page,
+	} ) => {
+		// Search for the group block and insert it.
+		const inserterButton = page.locator(
+			'role=button[name="Toggle block inserter"i]'
+		);
+
+		if (
+			( await inserterButton.getAttribute( 'aria-pressed' ) ) === 'false'
+		) {
```

Comment:
Since that we should have full control of the initial state of the editor, we shouldn't need to check if the inserter is opened here. It should always be closed by default.